### DISCUSSION TITLE
rabbitmq-management not working 3.9.8 (IE 11)

### DISCUSSION THREAD

**Discussion by LordR7** [rabbitmq/rabbitmq-server#3687] [Other]
The web management site does not work on the current RabbitMQ Release 3.9.8 with Internet Explorer 11 (site is empty). 3.9.7 works without problems. With Firefox the problem does not occur.
Maybe the issue is in:
dispatcher.js, Row: 194, column: 47

If i copy the dispatcher.js from Release 3.9.7 to 3.9.8 the site works again.

![grafik](https://user-images.githubusercontent.com/94077993/141186990-ee4b71f3-c723-4fbd-b95f-a45ac050e14f.png)

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment]
[Clear your browser's cache](https://www.rabbitmq.com/upgrade.html#rabbitmq-plugins-compatibility). I cannot reproduce.

**Reply by LordR7** [reply]
Clearing the cache does not fix the issue.

**Comment by michaelklishin** [comment]
Actually, there was a change to that file on [that exact line](https://github.com/rabbitmq/rabbitmq-server/commit/bfee99c1497520f6df988dd246b63e9e78ca65bf#diff-10eed3a7972de14b21f44e78f76f68a1f202d3fe446310dafea700c0aec71520R194) so there may be something to it.

**Comment by LordR7** [comment]
Some more Information:
I see the issue with IE 11 on Windows Server 2012R2 and Server 2016.
A new installation of RabbitMQ does not fix the issue (deleted RabbitMQ Data Folder AppData\Roaming\RabbitMQ)
The Debug Console on IE11 shows:
Syntax error (JavaScript)
dispatcher.js, Row: 194, column: 47

**Reply by michaelklishin** [reply]
IE 11 is a browser from 2013 that does not support lambdas. See #3688. IE 11 support by Microsoft will be discontinued in June 2022, while we adjusted our code to be compatible for `3.9.9` this time, I highly recommend using a different browser such as Edge, Chrome, or Firefox.
